distribution/lib/Standard/Database/0.0.0-dev/src/Internal/SQLite/SQLite_Type_Mapping.enso

```diff
@@ -109,6 +111,14 @@ type SQLite_Type_Mapping
         Nothing -> Nothing
         _ : Vector -> column_type_suggestions.map .get
 
+    ## PRIVATE
+       Creates a `Column_Fetcher` used to fetch data from a result set and build
+       an in-memory column from it, based on the given column type.
+    make_column_fetcher : SQL_Type -> Column_Fetcher
+    make_column_fetcher self sql_type =
+        value_type = self.sql_type_to_value_type sql_type
+        Column_Fetcher_Module.default_fetcher_for_value_type value_type
```

Comment:
tbh I was slightly skeptical at first of moving this method to the type mapping, but seeing it in action it looks like the right call. 👍